src/train/train_cluster.py

```diff
@@ -49,18 +47,17 @@ def load_model(save_path) -> gensim.models.Word2Vec:
     """
     filename = save_path
     model = gensim.models.Word2Vec.load(filename)
-
     print('Loading succeed')
-    for index, word in enumerate(model.wv.index2word):
+    for index,word in enumerate(model.wv.index2word):
         if index == 5:
             break
-        vec = ",".join(map(lambda i: str(i), model.wv[word]))
+        vec = ",".join(map(lambda i: str(i),model.wv[word]))
         print(f"word #{index}/{len(model.wv.index2word)} is {word}, vec = {vec}")
     return model
 
 
 if __name__ == "__main__":
-    languange_name = 'Chinese'
+    languange_name = 'English'
```

Comment:
We can make for other languages, right?
Is it We are accessing this with Static?
wille